packages/babel-preset-typescript/src/index.js

```diff
@@ -2,7 +2,15 @@ import { declare } from "@babel/helper-plugin-utils";
 import transformTypeScript from "@babel/plugin-transform-typescript";
 
 export default declare(
-  (api, { jsxPragma, allExtensions = false, isTSX = false }) => {
+  (
+    api,
+    {
+      jsxPragma,
+      allExtensions = false,
+      isTSX = false,
+      allowNamespaces = false,
```

Comment:
I would prefer not to set a default value here, and let the plugin handle it.

Reply:
Makes sense, done 👍